### PR TITLE
Solved Pangram Exercise

### PR DESCRIPTION
**Pull request by iuliagatea**

### CODE REVIEW

File: Ruby/pangram/pangram.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  def self.pangram?(sentence)
+    new(sentence).pangram?
+  end
+
+  def initialize(sentence)
+    @sentence = sentence
+  end
+
+  def pangram?
+    ('a'..'z').to_a == ('a'..'z').to_a.select { |letter| sentence.downcase.include? letter }
```
Comment (asarbu) [inline Ruby/pangram/pangram.rb]: what is this `('a'..'z').to_a == ('a'..'z').to_a` supposed to do ?

File: Ruby/pangram/pangram.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class Pangram
```
Comment (asarbu) [inline Ruby/pangram/pangram.rb]: I think ('a'..'z') could be extracted into a constant

File: Ruby/pangram/pangram.rb (hunk 1 above)
Comment (iuliagatea) [reply Ruby/pangram/pangram.rb]: I check if the array resulted after running include is the same as the initial array (the alphabet)

Comment (asarbu) [reply Ruby/pangram/pangram.rb]: I see now, but for me this isn't easy to read. 

What would you think about using an approach that isn't iterating over all the letters of the alphabet ? For this you could `-` between 2 arrays

Comment (asarbu) [reply Ruby/pangram/pangram.rb]: you could use `empty?` if you slightly change your above code (reject is an option, but not a must).

`('a'..'z').to_a.select { |letter| !sentence.downcase.include?(letter) }.empty?`

I was hoping you will propose something that will use the difference between arrays, and at the end checks the result.

Comment (iuliagatea) [reply Ruby/pangram/pangram.rb]: @asarbu something like this (ALPHABET - sentence.downcase.chars.sort.uniq).empty?  ?

Comment (iuliagatea) [reply Ruby/pangram/pangram.rb]: done :) Thanks

File: Ruby/pangram/pangram.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  def self.pangram?(sentence)
+    new(sentence).pangram?
+  end
+
+  def initialize(sentence)
+    @sentence = sentence
+  end
+
+  def pangram?
+    (ALPHABET - sentence.downcase.chars.sort.uniq).empty?
```
[the remark this answers is not in the document]
Comment (paucio) [reply Ruby/pangram/pangram.rb]: Well you're not displaying it anywhere so it doesn't matter how it looks.
Performance wise every thing costs processing time so if it's not needed then you should not call it.

[the remark this answers is not in the document]
Comment (iuliagatea) [reply Ruby/pangram/pangram.rb]: done

**Pull request merged**